packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
-import RootStore from 'Stores/index';
-import AppStore from 'Stores/app-store';
-import { connect } from 'Stores/connect';
-import ReactJoyride from 'react-joyride';
 import classNames from 'classnames';
-import LoadModal from 'Components/load-modal';
 import { DesktopWrapper, MobileWrapper } from '@deriv/components';
+import LoadModal from 'Components/load-modal';
 import SaveModal from 'Components/save-modal';
-import WorkspaceWrapper from './workspace-wrapper';
+import ReactJoyride from 'react-joyride';
+import AppStore from 'Stores/app-store';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import { blocksCoordinate } from '../../../../../bot-skeleton/src/scratch/hooks/block_svg';
+import { setColors } from '../../../../../bot-skeleton/src/scratch/hooks/colours';
```

Comment:
can we use alias to import from bot-skeleton?

Reply:
done ✅